src/compiler/types.ts

```diff
@@ -2797,6 +2772,8 @@ namespace ts {
         InArrayType                     = 1 << 15,  // Writing an array element type
         UseAliasDefinedOutsideCurrentScope = 1 << 16, // For a `type T = ... ` defined in a different file, write `T` instead of its value,
                                                       // even though `T` can't be accessed in the current scope.
+        MultilineObjectLiterals         = 1 << 17,   // Always print object literals across multiple lines (only used to map into node builder flags)

```

Comment:
which flags did the SymbolWriter use? I expected one enum to go away, and instead I see only additions to enums around here.

Reply:
The `SymbolWriter` used `SymbolFormatFlags` and `TypeFormatFlags`. Both are still in use as part of the public API of the `XToString` functions, however if you'd like I can replace them all with `NodeBuilderFlags` directly - it'd make the code a lot simpler, for sure.

Reply:
I don't have a good sense for how may people rely on our API not changing. I'd prefer to change it, but probably @mhegazy and @DanielRosenwasser have more data on how bad API breaks are. Thoughts?